nx-dev/ui-customers/src/lib/customer-testimonial-carousel.tsx

```diff
@@ -0,0 +1,400 @@
+          'Nx Cloud frees teams from managing CI, letting devs focus on code.',
+      },
+      {
+        value: 'Reduced duplication',
+        label:
+          'Shared libraries cut down redundant code across mobile and web apps.',
+      },
+    ],
+    company: 'UKG',
+    videoId: 'rSC8wihnfP4',
+    thumbnail: '/images/customers/video-story-ukg.avif',
+    logo: {
+      icon: UkgIcon,
+      height: 'h-20',
+      width: 'w-20',
+    },
+  },
+  {
+    title: 'Customer story',
+    subtitle: 'How Broadcom stays efficient and nimble with monorepos',
+    metrics: [
+      {
+        value: '2x faster',
+        label: 'Nx doubles speed, supporting fast, nimble development.',
+      },
+      {
+        value: 'Unmatched DX',
+        label: 'Nx empowers teams to be more productive than ever.',
+      },
+      {
+        value: 'Always ahead',
+        label:
+          'Nx’s rapid feature delivery keeps teams at the cutting edge of development.',
+      },
+    ],
+    company: 'Broadcom (VMware)',
+    videoId: 'RWTgYNKqxNc',
+    thumbnail: '/images/customers/video-story-broadcom.avif',
+    logo: {
+      icon: VmwareIcon,
+      height: 'h-28',
+      width: 'w-28',
+    },
+  },
+];
+
+export function CustomerTestimonialCarousel(): JSX.Element {
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [isOpen, setIsOpen] = useState(false);
+  const currentTestimonial = testimonials[currentIndex];
+  const slideLogoTimeOut = 12000; // 12 seconds
+
+  useEffect(() => {
+    let timer: NodeJS.Timeout;
+
+    if (!isOpen) {
+      timer = setInterval(() => {
+        setCurrentIndex((prevIndex) => {
+          const nextIndex = (prevIndex + 1) % testimonials.length;
+          return nextIndex;
+        });
+      }, slideLogoTimeOut);
+    }
+
+    return () => {
+      clearInterval(timer);
+    };
+  }, [currentIndex, setCurrentIndex, isOpen]);
+
+  return (
+    <div className="w-full">
+      <div className="mx-auto grid max-w-7xl grid-cols-2 gap-2 px-4 lg:grid-cols-5 lg:gap-4">
+        {/* Left side - Quote or Metrics */}
+        <div className="col-span-2 hidden lg:block">
+          <div className="flex h-full flex-col justify-center space-y-8">
+            {currentTestimonial.metrics?.map((metric, index) => (
+              <div key={index} className="space-y-2">
+                <div className="border-l-2 border-sky-900/70 pl-4 text-3xl font-bold text-slate-700 dark:border-blue-300/60 dark:text-slate-200">
+                  {metric.value}
+                </div>
+                <div className="text-balance pl-[18px] text-lg text-slate-500 dark:text-slate-400">
+                  {metric.label}
+                </div>
+              </div>
+            ))}
+          </div>
+        </div>
+
+        {/* Right side - Video Card */}
+        <div className="col-span-2 md:col-span-3">
+          <div className="flex items-center gap-4">
+            {/* Prev Button Mobile only */}
+            <button
```

Comment:
Add a text to the button describing the action set with sr-only for accessibility purpose.

Reply:
done